src/test/java/org/commcare/formplayer/tests/CaseClaimNavigationTests.java

```diff
@@ -1,388 +1,230 @@
 package org.commcare.formplayer.tests;
```

Comment:
Best to review this file without the diff: https://github.com/dimagi/formplayer/blob/4406a0f58c47bd72155d67d14217585615bff389/src/test/java/org/commcare/formplayer/tests/CaseClaimNavigationTests.java

Unfortunately after I renamed `CaseClaimNavigationParentChildTests.java` I re-created the file with the same name which has made the diff very confusing.